File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Convert Word to PDF in Windows Forms | Syncfusion 
+description: Convert Word document to PDF in Windows Forms application using Syncfusion .NET Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word document to PDF in Windows Forms
+
+Syncfusion DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to **create, read, edit and convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert Word document to PDF in Windows Forms**.
+
+## Steps to convert Word document to PDF programmatically:
+
+Step 1: Create a new Windows Forms application project.
+
+Step 2: Install [Syncfusion.DocIO.WinForms](https://www.nuget.org/packages/Syncfusion.DocIO.WinForms/) NuGet package as a reference to your Windows Forms application from the [NuGet.org](https://www.nuget.org/).
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering Syncfusion license key in your application to use our components.
+
+Step 3: Include the following namespaces in the **Form1.Designer.cs** file.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using Syncfusion.OfficeChart
+using Syncfusion.DocToPDFConverter
+using Syncfusion.OfficeChartToImageConverter
+using Syncfusion.Pdf
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 4: Add a new button in **Form1.Designer.cs** to create Word file as follows.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+private Button WordToPDF;
+private Label label;
+
+private void InitializeComponent()
+{
+    label = new Label();
+    WordToPDF = new Button();
+
+    //Label
+    label.Location = new System.Drawing.Point(0, 40);
+    label.Size = new System.Drawing.Size(426, 35);
+    label.Text = "Click the button to view a Word document generated by Essential DocIO. Please note that Microsoft Word Viewer or Microsoft Word is required to view the resultant Word document";
+    label.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
+
+    //Button
+    WordToPDF.Location = new System.Drawing.Point(180, 110);
+    WordToPDF.Size = new System.Drawing.Size(85, 36);
+    WordToPDF.Text = "Convert Word document to PDF";
+    WordToPDF.Click += new EventHandler(WordToPDF_Click);
+
+    //Create Word
```

Comment:
//Create Word -- change this to //Word to PDF

Reply:
Addressed